tests/connector-service-tests/test/apitests/connector_test.go

```diff
@@ -175,8 +177,8 @@ func TestConnector(t *testing.T) {
 		// then
 		// TODO after fixing the service, set status code to 400.
```

Comment:
Please remove TODO, as it is done in your PR

Reply:
Done